includes/apple-exporter/builders/class-components.php

```diff
@@ -184,11 +184,16 @@ private function _add_thumbnail_if_needed( &$components ) {
 				return;
 			}
 
-			// Use this image as the cover and remove it from the body to avoid
-			// duplication.
+			// Use this image as the cover.
 			$this->set_content_property( 'cover', $original_url );
-			unset( $components[ $i ] );
-			$components = array_values( $components );
+
+			// If the cover is set to be displayed, remove it from the flow.
+			$order = $this->get_setting( 'meta_component_order' );
+			if ( is_array( $order ) && in_array( 'cover', $order ) ) {
+				unset( $components[ $i ] );
```

Comment:
It's using tabs correctly in source. Not sure why it's rendering weird in this view.